### PR TITLE
Document the Data Model for MTL

### PR DESCRIPTION
**Pull request by geordie666**
This PR documents the data model for the MTL ledgers. 

As part of this PR, a new `$SURVEYOPS` directory at the root level of the data model is added to document files used by survey operations.

### CODE REVIEW

**Comment by geordie666** [comment]
This should address #97.

**Comment by geordie666** [comment]
In case it's helpful for reviewers, I've dumped the sphinx output here:

https://data.desi.lbl.gov/desi/users/adamyers/ADM-dm-MTL/html/

File: doc/SURVEYOPS/mtl/index.rst
Comment (weaverba137) [inline doc/SURVEYOPS/mtl/index.rst:10]: Are there meant to be other "done" files besides `(scnd-|)mtl-done-tiles\.ecsv`?  In other words, you could be a lot more specific than `*done*ecsv`.

File: doc/SURVEYOPS/ops/index.rst
Comment (weaverba137) [inline doc/SURVEYOPS/ops/index.rst:5]: Is the description of the ops directory beyond the scope of this PR?

**Review by weaverba137 — CHANGES_REQUESTED** [review]
This looks fine to me. A couple of minor questions that should be addressed before merging.

For the record, we currently don't have any support for *validating* .ecsv files, but it's something we need to do anyway and this PR provides much stronger motivation for that.

Related: ECSV files don't have the concept of multiple HDUs, at least in versions I've looked at, so in the future we might remove the Contents section. But let's not worry about that for this PR.

File: doc/SURVEYOPS/mtl/index.rst
Comment (geordie666) [reply doc/SURVEYOPS/mtl/index.rst]: Agreed. I updated this to read:

```
The ``mtl-done-tiles.ecsv`` and ``scnd-mtl-done-tiles.ecsv`` files in the ``mtl`` directory track which tiles...
```

File: doc/SURVEYOPS/ops/index.rst
Comment (geordie666) [reply doc/SURVEYOPS/ops/index.rst:5]: It is. Eddie is on the hook to describe the files in the `ops` directory. I thought it might help to include a placeholder for when Eddie is ready to take on that task.

**Review by weaverba137 — APPROVED** [review]

**Comment by weaverba137** [comment]
Thanx for the changes.  OK to merge when ready.

**Pull request merged**
